### DISCUSSION TITLE
Text Node (beta)

### DISCUSSION THREAD

**Discussion by zachowj** [zachowj/node-red-contrib-home-assistant-websocket#963] [General]
placeholder

**Comment by odis68** [comment]
Thanks for your work !!
There is no possibility of input data for the text of the node. Only a way out !!

![Скриншот 10-05-2024 16 20 50](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/assets/46837765/7ac7ca2b-a37e-404a-9c28-e07554a2d45b)
![Скриншот 10-05-2024 16 21 55](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/assets/46837765/3b787fcb-fa7c-4db3-b7e3-4af5e94653d0)
![Скриншот 10-05-2024 16 23 06](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/assets/46837765/a771e756-3f47-4eb5-b09b-415bc210056f)

**Reply by zachowj** [reply]
Change the mode to `set value`

**Comment by odis68** [comment]
Thank you!!!!
Your answer helped.
It was not clear to me.

**Comment by quizzical86** [comment]
@zachowj 
I have a few of these text nodes that set the value of a text entity.
I have the impression that when restarting Home Assistant, during the restart, the text entity gets the status "unavailable".
Then once Home Assistant has loaded again, the text nodes will get status "registered" and one of them will write the value I have configured it with into the text sensor.
![image](https://github.com/user-attachments/assets/a3b6d8d3-bb7d-475c-a065-0e2122eedbe9)

My workaround to control what value gets written when Home Assistant is back up and running, is to have Node Red detect when Home Assistant has restarted, then pause for a few seconds (to let the rather random behaviour described above take place first), and then after the pause write the value I want to the text entity.

**Comment by dr-carrot** [comment]
Can we add attributes to this too?
